modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -206,6 +206,59 @@ module_stream_v2_test_profiles (ModuleStreamFixture *fixture,
 }
 
 
+static void
+module_stream_v1_test_rpm_artifacts (ModuleStreamFixture *fixture,
+                                     gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStreamV1) stream = NULL;
+  g_auto (GStrv) artifacts = NULL;
+
+  stream = modulemd_module_stream_v1_new (NULL, NULL);
+
+  modulemd_module_stream_v1_add_rpm_artifact (
+    stream, "bar-0:1.23-1.module_deadbeef.x86_64");
+  artifacts = modulemd_module_stream_v1_get_rpm_artifacts_as_strv (stream);
+  g_assert_true (g_strv_contains ((const gchar *const *)artifacts,
```

Comment:
Please also check that length of this is equal to 1.

Reply:
I have added this check.